### PR TITLE
added snuffleupagus ext support

### PR DESCRIPTION
**Pull request by LAKostis**
Added [snuffleupagus](https://snuffleupagus.readthedocs.io/) extension.

Small changes:
- replace get php version func with awk (simpler).
- check for src/ dir existence in module archive.

### CODE REVIEW

File: install-php-extensions [hunk 1]
```diff
@@ -55,7 +55,7 @@ getDistroVersion() {
 # Output:
 #   The PHP numeric Major-Minor version
 getPHPMajorMinor() {
-	php -r '$v = explode(".", PHP_VERSION); echo $v[0] * 100 + $v[1];'
+	php-config --version | awk -F. '{print $1*100+$2}'
```
Comment (mlocati) [inline install-php-extensions]: Any reason for this change?

Comment (LAKostis) [reply install-php-extensions]: awk works faster for the first run (as php reads up all libraries/dependencies):

$ docker run -ti lakostis123/php7-fpm-alpine /bin/sh
/var/www/html # time php -r '$v = explode(".", PHP_VERSION); echo $v[0] * 100 + $v[1];'
703real 0m 0.07s
user    0m 0.01s
sys     0m 0.01s

/var/www/html # time php-config --version | awk -F. '{print $1*100+$2}'
703
real    0m 0.00s
user    0m 0.00s
sys     0m 0.00s

But of course it's up to you to decide.

**Comment by mlocati** [comment]
If my changes are ok for you, I'd merge this PR

**Comment by LAKostis** [comment]
Yep, I'm okay with those changes. Thanks for your assistance!

**Pull request merged**